### PR TITLE
json_to_abap - added logic for type time

### PR DESCRIPTION
**Pull request by oblomov-dev**
never used regex before, but tried to make it as similar as possible to the already existing date logic.

### CODE REVIEW

File: src/core/zcl_ajson.clas.testclasses.abap [hunk 1]
```diff
@@ -1457,7 +1457,9 @@ class ltcl_json_to_abap definition
     methods to_abap_corresponding_pub_neg
       for testing
       raising zcx_ajson_error.
-
+    methods to_abap_time
+                FOR TESTING
+      reaising cx_static_check.
```
Comment (larshp) [inline src/core/zcl_ajson.clas.testclasses.abap:1462]:
```suggestion
      raising cx_static_check.
```

Comment (oblomov-dev) [reply src/core/zcl_ajson.clas.testclasses.abap]: fixed!

File: src/core/zcl_ajson.clas.testclasses.abap [hunk 2]
```diff
@@ -1457,7 +1457,9 @@ class ltcl_json_to_abap definition
     methods to_abap_corresponding_pub_neg
       for testing
       raising zcx_ajson_error.
-
+    methods to_abap_time
+                for testing
```
Comment (sbcgua) [inline src/core/zcl_ajson.clas.testclasses.abap:1461]: Could you pls indent it to the left (as above).

P.S. sorry, I'm a bit picky about readability (indentation in particular)

File: src/core/zcl_ajson.clas.testclasses.abap [hunk 3]
```diff
@@ -1534,6 +1536,28 @@ class ltcl_json_to_abap implementation.
...
+    create object lo_cut.
+    lo_cut->to_abap(
+      exporting
+        it_nodes    = lo_nodes->sorted( )
+      changing
+        c_container = lv_mock ).
+
+    cl_abap_unit_assert=>assert_equals(
+      act = lv_mock
+      exp = '111111' ).
```
Comment (sbcgua) [inline src/core/zcl_ajson.clas.testclasses.abap:1557]: Please add a test with empty value. It should work accroding to the code, but better to have the test. In the same method would be just OK.

**Review by sbcgua — CHANGES_REQUESTED** [review]
Looks good, just a couple of polishings

**Comment by oblomov-dev** [comment]
thank you for reviewing!

**Review by sbcgua — APPROVED** [review]
Nice. Merging. Thanks for the catch and fix

P.S. will see the perf issues later, seems they are not 5 min answers ...

**Comment by sbcgua** [comment]
@larshp abaplint UTs are failing, looks like different `initial time` behavior. Any quick suggestion? (exclude is an option too)

**Comment by larshp** [comment]
empty value will trigger the exception right? that is then ignored and initial values given?

**Comment by larshp** [comment]
fixed, running

**Comment by sbcgua** [comment]
Thanks Lars !

**Comment by sbcgua** [comment]
Using the occasion, v1.1.10
